tests/cases/fourslash/codeFixInitializePrivatePropertyJS.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference path='fourslash.ts' />
+
+// @allowjs: true
+// @checkJs: true
+
+// @Filename: /a.js
+////class Foo {
+////    constructor(name) {
+////        this.[|#name|] = name;
+////    }
+////}
+
+verify.codeFixAvailable([
+    { description: "Initialize a private property '#name'" },
+    { description: "Ignore this error message" },
+    { description: "Disable checking for this file" },
+    { description: "Infer parameter types from usage" },
+]);
+
+verify.codeFix({
+    index: 0,
+    description: "Initialize a private property '#name'",

```

Comment:
Can you reference `ts.DiagnosticMessages.[...].text` here?

Reply:
@DanielRosenwasser done